sdk/resourcemanager/README.md

```diff
@@ -28,7 +28,7 @@ If you are an existing user of the older version of Azure management library for
 <dependency>
   <groupId>com.azure.resourcemanager</groupId>
   <artifactId>azure-resourcemanager</artifactId>
-  <version>2.0.0-beta.3</version>
+  <version>2.0.0-beta.4</version>
```

Comment:
As doc, it appears better to use `dependency` instead of `current`, though latter is widely used for other packages.

Reply:
Sorry didn't get this. For doc, user normally copy/paste, so current.

Reply:
current is not released (beta.4 here).

Reply:
got it. `current` we're talking about isn't same thing :) . `current` for me is latest released version. In original context of the comment, it's current in development version. While our believe is same: let's use the latest released one: 2.0.0-beta.3.

Reply:
I will use beta.3 here. @ChenTanyi will figure it out in next release (maybe the "current" in md is different from "current" in pom).